src/pages/elements/Service.js

```diff
@@ -21,29 +21,84 @@ const linkFormat = url => value => (
 const emptyFormat = value => value || '-';
 const booleanFormat = (t, f) => value => (value ? t : f);
 
+// link to window.DATA_DIR_URL path 
```

Comment:
I think we should put the following part in components folder.

Reply:
Thanks for your feedback! I've moved them to NonEmptyDefinition.js/ NamePathList.js under components.